data/aliases.js

```diff
@@ -428,6 +432,25 @@ exports.BattleAliases = {
 	"ohmagod": "Plasmanta",
 	"zong": "Bronzong",
 
+	// ultra beasts
+	"absorption": "Buzzwole",
+	"beauty": "Pheromosa",
+	"blade": "Kartana",
+	"blaster": "Celesteela",
+	"expansion": "Buzzwole",
+	"gluttony": "Guzzlord",
```

Comment:
The English name for it is Glutton, which is lucky for us since Gluttony is already a thing.